widgets/pagination.js

```diff
@@ -1,6 +1,8 @@
 var React = require('react');
 
 var utils = require('../lib/utils.js');
+var autoHide = require('../decorators/autoHide');
+var Pagination = autoHide(require('../components/Pagination/Pagination.js'));
```

Comment:
Mixed feelings, on one hand this is very descriptive, on another hand this is a lot of letters for a variable name. And I fear we end up with `RefinementListHeaderFooterAutoHideAndSomeOtherBehaviorsThatWeWillHate`.

As for determining what are the behaviors of `<Pagination>` right now you just have to the top and see what are the decorators, should not be something we need to deal with a lot of times.